sdk/src/error.rs

```diff
@@ -241,7 +241,7 @@ pub enum Error {
     CborError(#[from] serde_cbor::Error),
 
     #[error(transparent)]
-    #[cfg(feature = "file_io")]
+    #[cfg(feature = "sign")]
     OpenSslError(#[from] openssl::error::ErrorStack),
```

Comment:
Scoping this to `sign` feature seems odd to me. Do we not also use OpenSSL for verifying signatures?

Reply:
Apparently we don't us OpensslError for verify, so I'd need to allow an unused reference if I don't scope it this way.

Reply:
I'll see if feature = "openssl" will work.

Reply:
We were disabling openssl when not using file_io because we don't use openssl with wasm.